.github/workflows/CICD.yml

```diff
@@ -475,6 +475,12 @@ jobs:
         ## Install `rust` toolchain
         rustup toolchain install nightly --profile minimal
         rustup default nightly
+    - name: Install/setup prerequisites
+      shell: bash
+      run: |
+        case '${{ matrix.job.os }}' in
+          ubuntu-latest) sudo apt-get -y update ; sudo apt-get -y install fuse libfuse-dev pkg-config ;; # needed for fuser 
```

Comment:
can you be more descriptive in your comment?
(why do we use fuser?)

Reply:
yes, I will cleanup the comments after I get the CI to work